src/Datastore/DatastoreSessionHandler.php

```diff
@@ -204,9 +216,21 @@ public function read($id)
 
     /**
      * Write the session data to Cloud Datastore.
+     *
+     * @param string $id Identifier used to construct a {@see \Google\Cloud\Datastore\Key} for the {@see \Google\Cloud\Datastore\Entity} to be written.
+     * @param string $data The session data to write to the {@see \Google\Cloud\Datastore\Entity}.
+     * @param array $entityOptions Optional arguments that will be passed to the {@see \Google\Cloud\Datastore\DatastoreClient::entity} method
+     * @return bool
      */
-    public function write($id, $data)
+    public function write($id, $data, $entityOptions = null)
```

Comment:
Good point - I have removed this, so that the entityOptions can only be set in the constructor.